src/flb_config.c

```diff
@@ -345,6 +345,11 @@ struct flb_config *flb_config_init()
     mk_list_init(&config->cmetrics);
     mk_list_init(&config->cf_parsers_list);
 
+    /* Initialize multiline-parser list. We need this here, because from now
```

Comment:
Not strictly related to my change. I got an unrelated segfault fuzz failure in CI and this fixes the issue.

Reply:
Let's isolate this in another PR, I've already approved yours as it was and don't want to mix things up so please revert this change and open a new one with the appropriate context so your reasoning is easy to follow and validate.